apps/tasks/lib/screens/my_tasks_screen.dart

```diff
@@ -52,9 +53,49 @@ class _MyTasksScreenState extends State<MyTasksScreen> {
                         )
                     ),
                     key: ValueKey(_items[index]),
-                    child: TaskListTile(
-                      index: index,
-                      task: _items[index],
+                    child: Dismissible(
+                      confirmDismiss: (DismissDirection direction) async {
+                        if (direction == DismissDirection.endToStart) {
+                          // TODO: replace with API-Call
+                          return true;
+                        }
+                        else{
+                          // TODO: mark task as done
```

Comment:
```suggestion
                          // TODO: add API Call
                          _items[index]['isChecked'] = true;
```

Reply:
Could also be `_items[index]['isChecked'] = !_items[index]['isChecked'], but this would require a different dimissable background for endToStart